netket/graph/lattice.py

```diff
@@ -22,6 +22,8 @@
 
 
```

Comment:
```suggestion
cutoff_tol = 1e-5
"""Tolerance for the maximum distance cutoff when computing the sparse distance matrix.
This is necessary because of floating-point errors when computing the distance in non-trivial 
lattices.
""" 

```

Reply:
Add this here and use it below as needed.
This way it is more clear why we have it and we always use the same number (easier to change in the future if necessary).

Reply:
Yeah this makes sense